src/cli/cli.cpp

```diff
@@ -7802,13 +7802,48 @@ template <> otError Interpreter::Process<Cmd("mac")>(Arg aArgs[])
 {
     otError error = OT_ERROR_NONE;
 
+    /**
+     * @cli mac retries direct (get,set)
```

Comment:
I would move this block before the `if (aArgs[1] == "direct")` line since it's specific to the `direct` arg, just like you did for the `indirect` one